bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -2,201 +2,135 @@
-              </ul>
-            </div>
-            <div class="fxos-latest-links-wrapper latest-links-wrapper">
-              <ul>
-                <li><a class="more" href="{{ url('firefox') }}">{{_('Discover all the ways to take Firefox on the go')}}</a></li>
-              </ul>
-            </div>
-
-            <div id="other-platforms-languages-wrapper" class="hidden">
-              <button id="other-platforms-modal-link" type="button">
-              {% if l10n_has_tag('firefox_new_install_options') %}
-                {{ _('Advanced install options & other platforms') }}
-              {% else %}
-                {{ _('Firefox for other platforms') }}
-              {% endif %}
-              </button>
-
-              <p>
-                <a href="{{ url('firefox.all') }}">
-                {% if l10n_has_tag('firefox_new_install_options') %}
-                  {{ _('Other languages') }}
-                {% else %}
-                  {{ _('Available Languages') }}
-                {% endif %}
-                </a>
-              </p>
-            </div>
-
-            <section id="benefits">
-              <ul>
-                <li>
-                  <h2 class="fast">{{_('Go Fast') }}</h2>
-                  <p>{{_('You have things to do. Get them done faster.') }}</p>
-                </li>
-                <li>
-                  <h2 class="far">{{_('Go Far') }}</h2>
-                  <p>{{_('Go anywhere, with more built-in privacy tools than any other browser.') }}</p>
-                </li>
-                <li>
-                  <h2 class="forward">{{_('Go Forward') }}</h2>
-                  <p>{{_('Use the only browser built for people, not profit.') }}</p>
-                </li>
-              </ul>
-            </section>
-          </div>{#-- /.main-content --#}
-        </div>{#-- /.inner-container --#}
-      </div>
-    </div>
+{% block messages %}
+<div class="version-message-container">
+  <div class="content">
+    <p class="version-message firefox-latest">{{_('Congrats! You’re using the latest version of Firefox.') }}</p>
+    <p class="version-message android-old">{{_('<a href="%(url)s">Update</a> your Firefox for the latest in speed and privacy.')|format(url='https://support.mozilla.org/kb/update-latest-version-firefox-android')}}</p>
+    <p class="version-message desktop-old">{{_('<a href="%(url)s">Update</a> your Firefox for the latest in speed and privacy.')|format(url='https://support.mozilla.org/kb/update-firefox-latest-version') }}</p>
+    <p class="version-message firefox-pre-release">{{_('You’re using a pre-release version of Firefox.') }}</p>
   </div>
-  <div class="foreground">
-    <aside id="newsletter-subscribe">
-      <div class="container">
-        {{ email_newsletter_form(title=_('Keep up with<br> all things Firefox.'), button_class='button-light', spinner_color='#fff') }}
+</div>
+{% endblock %}
+
+{% block main_content %}
+<header>
+{% if l10n_has_tag('firefox_quantum_new_headline') %}
```

Comment:
I think we might be able to drop this conditional? Looks like coverage is good for [both](https://l10n.mozilla-community.org/langchecker/?website=0&file=firefox/new/quantum.lang&action=translate&show#string33) [strings](https://l10n.mozilla-community.org/langchecker/?website=0&file=firefox/new/quantum.lang&action=translate&show#string34).

Reply:
Good call 👍